skill-tree /src/main/java/com/RDS/skilltree/Skill/SkillsController.java

```diff
@@ -39,9 +40,9 @@ public ResponseEntity<?> createSkill(@RequestBody(required = true) @Valid SkillD
 
     @GetMapping("/")
     public Page<SkillDTO> getAllSkills(
-            @RequestParam(value = "page", defaultValue = "0") int page,
-            @RequestParam(value = "size", defaultValue = "100") int size) {
-        Pageable pageable = PageRequest.of(page, size);
+            @RequestParam(value = "offset", defaultValue = "0", required = false) @Min(0) int offset,
```

Comment:
what's the point of @Min(0) if required is false

Reply:
Its for the validation that you cannot have < 0 for the input.
Required says its mandatory or optional

Reply:
got it